lenet/lenet.js

```diff
@@ -176,7 +176,9 @@ export class LeNet {
   }
 
   async build(outputOperand) {
-    this.graph_ = await this.builder_.build({'output': outputOperand});
+    if (this.builder_) {
```

Comment:
Why add this builder check here?

Reply:
Good catch, there was an error but it was gone when not click the backend buttons too early (when DOMContentLoaded). Removed this check in https://github.com/webmachinelearning/webnn-samples/pull/281/commits/0d03980aafbf5b765affe63522f8c65bc9329189 @Honry Thanks!